packages/platform-sdk-profiles/src/profiles/aggregates/transaction-aggregate.ts

```diff
@@ -39,7 +39,11 @@ export class TransactionAggregate {
 			.includes(true);
 	}
 
-	public flush(): void {
+	public flush(method: string): void {
```

Comment:
Make this an optional parameter and either flush all history or a specific history depending on if it is provided or not.